### ISSUE TITLE
Factory Functions and the Module Pattern: Reword a step in the Assignment section

### ISSUE THREAD

**Issue by T0nci** [TheOdinProject/curriculum#27484]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

A suggestion to reword Step 4 in the Assignment section to make it more clear that this step is optional if you have read the above articles.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-factory-functions-and-the-module-pattern

### (Optional) Discord Name

T0nci

**Comment by MaoShizhong** [comment]
@T0nci  I think this a little extra clarity wouldn't hurt, given it seems to be clearly intended as an optional alternative and the content does cover the same concepts as previous resources/assignments.

I would actually also like for a little extra to be added to that assignment step to mention that while the videos include JQuery syntax, the focus is on the concepts of objects and the module pattern (where you don't really need to understand the JQuery syntax specifically).

Would you be happy to include something like this in addition to the change about clarity on the step being an optional alternative? If so, I can assign this to you to work on.

**Comment by T0nci** [comment]
@MaoShizhong Yes, of course ^^! I would be more than happy to work on this issue and also mention the use of jQuery + "but that shouldn't scare you away since you don't need to understand the jQuery syntax, only focus on the previously learnt concepts".

Should I reword it to be as an alternative or as a supplement if it's not fully understood?

**Comment by MaoShizhong** [comment]
@T0nci Something like
> As an optional alternative, in case you prefer video lessons ...

should probably suffice.

**Comment by MaoShizhong** [comment]
Oops, wrong button - *you're* the one assigned now, not me :stuck_out_tongue_closed_eyes:

**Comment by T0nci** [comment]
@MaoShizhong Why is it that all the items in the bulleted list are 1s? While checking on the site, they are different numbers, but in the md file they are not. Or is it just my editor/fork? (for reference, this is the md file for the Factory Functions and the Module Pattern lesson)
![image](https://github.com/TheOdinProject/curriculum/assets/138928016/a3c93d6d-04f8-4910-8bd5-112af78e637c)

**Comment by T0nci** [comment]
![image](https://github.com/TheOdinProject/curriculum/assets/138928016/0670920c-a4fd-4ad3-9cec-f89ab2976353)

**Comment by MaoShizhong** [comment]
@T0nci That's part of the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lazy-numbering) where we use lazy numbering to make it easier to add/remove list items in the future without having to manually edit the numbers. Markdown is smart enough to figure things out as long as we indent appropriately.
